README.md

```diff
@@ -1,46 +1,27 @@
 # Aam Digital Setup
 This repository describes how to set up everything that is needed to run Aam Digital in production.
-This includes deploying the app, deploying and connecting the database and optionally deploying and connecting the permission backend.
+This includes deploying the app, deploying and connecting the database and optionally deploying and connecting the permission backend and keycloak.
 
-## Deploying without permission backend (simple)
-To set up the application with a direct connection to the database, without a permission backend, follow these steps:
+## Deploying the application
+The interactive script `interactive_setup.sh` walks you through the process of setting up the application.
+Just run it and look for the console output and questions.
 
-1. Clone this repository to easily get all the setup files: `git clone https://github.com/Aam-Digital/ndb-setup.git`
-2. Edit the `docker-compose.yml`
-   1. in `app` set `VIRTUAL_HOST` and `LETSENCRYPT_HOST` to the desired URL.
-   2. in `couchdb` set `COUCHDB_PASSWORD` to secure password. This is used for admin tasks on the database.
-3. Create a `config.json` file that overwrites the `session_type` and `demo_mode` settings:
-   ```json
-    {
-      "session_type": "synced",
-      "demo_mode": false
-    }
-   ```
-4. Run `docker-compose up -d`
-5. Run `./initial-setup.sh https://<VIRTUAL_HOST> <COUCHDB_PASSWORD>`
-6. To add users run `js add-user.js <VIRTUAL_HOST> admin:<COUCHDB_PASSWORD> <USERNAME> <PASSWORD>` where `USERNAME` and `PASSWORD` are the desired user credentials.
+The following things can be automatically done
 
+1. Deploy the application
+2. (optional) add the permission backend
+3. (optional) connect with a running Keycloak
+4. (optional) migrate users from CouchDB to Keycloak
 
-## Deploying with permission backend (advanced)
-Use this deployment if you want to enable permission checks in your application
+To log errors with [Sentry](https://sentry.io/), simply set the variable `SENTRY_DSN` in the `.env` file to you sentry DSN and restart the container (`docker compose down && docker compose up -d`).
```

Comment:
to which .env file? the one here in the keycloak folder? one that you have to create in the root folder? 

And is it not using the docker-compose anymore?

Reply:
The `.env` file in the top level folder is automatically created during the setup process (I added that info to the readme). What do you mean by docker compose? I have moved all the variables that are changed for starting a new instance to the separate `.env` file. This will make it easier to update the `docker-compose.yml` using normal git operation as this file does not need to be modified.

Reply:
Generally, I am also not to sure whether it would actually be better to have these two thing separated (the setup repo that contains templates and script and the folder where the application is running) what do you think?